app/components/pull-request-page.js

```diff
@@ -109,22 +109,25 @@ export default Ember.Component.extend({
           "standardSet.jurisdiction.title",
         ]
 
-        if (!_.includes(fieldsToDetermineValidate, errorObj[0].key)) {
-          set(this, "descriptionIsValid", true)
-        }
+        let allErrorKeys = _.map(errorArray, error => {
+          return error.key
+        })
```

Comment:
I'd put this on one line in the future, but no need to change it if that's all you need to do.